### PR TITLE
[v2] - Refactor Errors Component

### PR DESCRIPTION
**Pull request by joaopalopes24**
Refactor Errors Component.

### CODE REVIEW

File: src/lang/en/messages.php [hunk 1]
```diff
@@ -13,6 +13,6 @@
         'yesterday' => 'Yesterday',
     ],
     'errors' => [
-        'title' => 'There were {errors} errors with your submission',
+        'title' => 'There was {errors} error with your submission|There were {errors} errors with your submission',
```
Comment (ph7jack) [inline src/lang/en/messages.php]:
```suggestion
        'title' => 'There was an error with your submission|There were 3 errors in your submission',
```

File: src/lang/pt_BR/messages.php [hunk 2]
```diff
@@ -13,6 +13,6 @@
         'yesterday' => 'Ontem',
     ],
     'errors' => [
-        'title' => 'Foram encontrados {errors} erros de validação',
+        'title' => 'Foi encontrado {errors} erro de validação|Foram encontrados {errors} erros de validação',
```
Comment (ph7jack) [inline src/lang/pt_BR/messages.php]:
```suggestion
        'title' => 'Foi encontrado um erro de validação|Foram encontrados {errors} erros de validação',
```

File: src/View/Components/Card.php [hunk 3]
```diff
@@ -8,17 +8,23 @@
...
+
+    protected string $borderColor = 'border-secondary-200 dark:border-secondary-600';
+
     public function __construct(
         public ?string $title = null,
         public ?string $padding = null,
         public ?string $shadow = null,
         public ?string $rounded = null,
         public ?string $color = null,
+        public ?bool $undivided = null,
```
Comment (ph7jack) [inline src/View/Components/Card.php]: Can it be borderless?
```suggestion
        public ?bool $borderless = null,
```

File: src/View/Components/Errors.php [hunk 4]
```diff
@@ -3,54 +3,53 @@
...
-     * @param string $title
-     * @param string|array|null $only
-     */
     public function __construct(
-        ?string $title = null,
-        $only = [],
+        public ?string $title = null,
+        public mixed $only = [],
+        public ?string $icon = null,
+        public ?bool $undivided = null,
```
Comment (ph7jack) [inline src/View/Components/Errors.php]: Same as above

File: src/config.php [hunk 5]
```diff
@@ -1,51 +1,25 @@
...
-        'shadow'  => 'shadow-md',
-        'rounded' => 'rounded-lg',
-        'color'   => 'bg-white dark:bg-secondary-800',
+    'alert' => [
+        'padding'   => 'pl-1 mt-2 ml-5',
+        'shadow'    => 'shadow-md',
+        'rounded'   => 'rounded-lg',
+        'color'     => null,
+        'values'    => Alerts\Values::class,
+        'undivided' => false,
```
Comment (ph7jack) [inline src/config.php]: Same as above

File: src/config.php [hunk 6]
```diff
@@ -97,6 +74,51 @@
...
+    |
+    | The default card preferences
+    |
+    */
+    'card' => [
+        'padding'   => 'px-2 py-5 md:px-4',
+        'shadow'    => 'shadow-md',
+        'rounded'   => 'rounded-lg',
+        'color'     => 'bg-white dark:bg-secondary-800',
+        'undivided' => false,
```
Comment (ph7jack) [inline src/config.php]: Same as above

**Review by ph7jack — CHANGES_REQUESTED** [review]
Awesome!!! Can you sort out my comments?

File: tests/Unit/View/Components/SelectTest.php [hunk 7]
```diff
@@ -8,12 +8,12 @@
 class SelectTest extends TestCase
 {
     /** @test */
-    public function it_should_throw_an_error_if_async_data_and_options_is_set_together()
```
Comment (ph7jack) [inline tests/Unit/View/Components/SelectTest.php]: Instead of commenting out the test, use the `markTestSkipped` function to flag it as skipped.

Comment (joaopalopes24) [reply tests/Unit/View/Components/SelectTest.php]: I fix this test in PR https://github.com/wireui/wireui/pull/638/files.

Comment (joaopalopes24) [reply tests/Unit/View/Components/SelectTest.php]: And I had already done the Laravel Upgrade to version 10 in this PR too.

**Review by ph7jack — APPROVED** [review]
Excellent!!! Clean as water

**Pull request merged**